src/Migration.php

```diff
@@ -1220,6 +1220,74 @@ public function addRight($name, $rights = ALLSTANDARDRIGHT, $requiredrights = ['
         );
     }
 
+    /**
+     * Update right to profiles that match rights requirements
+     *    Default is to give rights to profiles with READ and UPDATE rights on config
+     *
+     * @param string  $name   Right name
+     * @param integer $rights Right to set (defaults to ALLSTANDARDRIGHT)
+     * @param array   $requiredrights Array of right name => value
+     *                   A profile must have these rights in order to get the new right.
+     *                   This array can be empty to add the right to every profile.
+     *                   Default is ['config' => READ | UPDATE].
+     *
+     * @return void
+     */
+    public function updateRight($name, $rights = ALLSTANDARDRIGHT, $requiredrights = ['config' => READ | UPDATE])
```

Comment:
We should not have a default here. It make sense for the `addRight` method, but it is dangerous to do it on update process.

```suggestion
    public function updateRight($name, $rights, $requiredrights = ['config' => READ | UPDATE])
```